app/controllers/concerns/administrate/punditize.rb

```diff
@@ -2,7 +2,12 @@ module Administrate
   module Punditize
     if Object.const_defined?("Pundit")
       extend ActiveSupport::Concern
-      include Pundit::Authorization
+
+      if Pundit.const_defined?(:Authorization)
+        include Pundit::Authorization
+      else
+        include Pundit
```

Comment:
This includes was deprecated in 2.2.0: https://github.com/varvet/pundit/blob/main/CHANGELOG.md#deprecated